.pylintrc

```diff
@@ -75,8 +73,9 @@ disable=missing-docstring,
         #TODO: Remove this once todos are done
         fixme,
         consider-using-with,
-        consider-using-f-string
-
+        consider-using-f-string,
+        # TODO: Remove this once update the code base
+        unnecessary-lambda-assignment
```

Comment:
Many lines raise an error on pylint:
```
arviz/tests/base_tests/test_stats_utils.py:108:11: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
arviz/tests/base_tests/test_stats_utils.py:118:11: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
```
So add this to disable section, and remove once updated the codebase, I could open an issue to reflect this.